tests/pytests/test_aggregate.py

```diff
@@ -659,7 +659,7 @@ def testModulo(self):
         # With Floats
         res = self.env.cmd('ft.aggregate', 'games', '*',
                                        'APPLY', '547758.3 % 5.1')
-        self.env.assertEqual(res[1][1], '3')
+        self.env.assertEqual(res[1][1], '3.00000000008')
```

Comment:
How did this change?

Reply:
previously, the modulo operation was performed on the rounded-down value of the numbers. We now use `fmod` to perform the floating point modulo operation, so we get a different value in this case.
The previous behavior can still be achieved with `floor(547758.3) % floor(5.1)`